package.json

```
@@ -144,6 +144,12 @@
           "default": "",
           "markdownDescription": "Pass additional arguments to the language server."
         },
+        "haskell.envVars": {
```

what about 
```suggestion
        "haskell.serverEnvironment": {
```
?

in the cmake extension it is named `environment` but prefixing it with server we make clear we are using for it and the option would be showed near to `haskell.serverExtraArgs`